src/Elements/TechCard/TechCard.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import classes from "./TechCard.module.css";
+
+const TechCard = (props) => {
+  return (
+    <div>
+      <div className={classes.card}>
+        <div className={classes.icon}>{props.icon}</div>
+        <div className={classes.heading}>{props.heading}</div>
```

Comment:
it would make more semantic sense if this was a paragraph element rather than a div